src/plots/hoverformat_attributes.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var FORMAT_LINK = require('../constants/docs').FORMAT_LINK;
+
+function axisHoverFormat(str) {
+    return {
+        valType: 'string',
+        dflt: '',
+        editType: 'none',
+        description: [
+            'Sets the hover text formatting rule for the ' + str + ' using d3 formatting mini-languages',
```

Comment:
need a more descriptive thing here explaining when this is used and what it overrides

Reply:
FYI - The current `zhoverformat` is generalized here.

Reply:
at least x, y, and z also support dates, right? But u, v, w I presume don't. Also as @nicolaskruchten mentioned we should say that the default behavior is `axis.hoverformat` (for x, y, z) and I'm not sure what for u, v, w - some generic number format I guess? Anyway it seems like the two flavors should have somewhat different forms of description.

Reply:
ah I guess `zhoverformat` has an axis (and supports dates) in 3D but for 2D it looks like u, v, w - so I guess it needs to be created in both flavors.